### PR TITLE
add ConsoleSize option

### PR DESCRIPTION
**Pull request by Mashimiao**
Signed-off-by: Ma Shimiao <mashimiao.fnst@cn.fujitsu.com>

### CODE REVIEW

File: cmd/oci-runtime-tool/generate.go [hunk 1]
```diff
@@ -79,6 +79,7 @@ var generateFlags = []cli.Flag{
 	cli.StringSliceFlag{Name: "process-cap-add", Usage: "add Linux capabilities"},
 	cli.StringSliceFlag{Name: "process-cap-drop", Usage: "drop Linux capabilities"},
 	cli.BoolFlag{Name: "process-cap-drop-all", Usage: "drop all Linux capabilities"},
+	cli.StringFlag{Name: "process-consolesize", Usage: "specifies the console size in characters"},
```
Comment (q384566678) [inline cmd/oci-runtime-tool/generate.go]: I think it's better to add an instance description like "e.g. WIDTH:HEIGHT" at the end.

Comment (Mashimiao) [inline cmd/oci-runtime-tool/generate.go]: Fine

**Comment by q384566678** [comment]
LGTM

**Comment by liangchenye** [comment]
LGTM

**Pull request merged**